fe1-web/parts/lao/organizer/eventCreation/events/CreateElection.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import {
+  View, Platform, TextInput, ScrollView,
+} from 'react-native';
+import 'react-datepicker/dist/react-datepicker.css';
+import { useNavigation } from '@react-navigation/native';
+
+import STRINGS from 'res/strings';
+import DatePicker from 'components/DatePicker';
+import ParagraphBlock from 'components/ParagraphBlock';
+import WideButtonView from 'components/WideButtonView';
+import { Timestamp } from 'model/objects';
+import TextBlock from 'components/TextBlock';
+import DropdownSelector from 'components/DropdownSelector';
+
+function dateToTimestamp(date: Date): Timestamp {
+  return new Timestamp(Math.floor(date.getTime() / 1000));
+}
```

Comment:
This function is replicated three times in the project (once here, once in Carlo's review, and once in one of the original DatePicker files). A solution could be adding this directly in the `Timestamp` mode/object file. You could have:
- `dateToTimestamp` (as a  static method)
- `timestampToDate` (since Carlo needs it and implements it in his PR) as a regular method

Reply:
Very good idea! Thank you
Implemented your suggestion in commit `f04fa372fba889bb92f7ec416286f1c05f621b0c`